drift-api/src/main/java/io/airlift/drift/TApplicationException.java

```diff
@@ -44,7 +44,8 @@
         PROTOCOL_ERROR(7),
         INVALID_TRANSFORM(8),
         INVALID_PROTOCOL(9),
-        UNSUPPORTED_CLIENT_TYPE(10);
+        UNSUPPORTED_CLIENT_TYPE(10),
+        TOO_LARGE_REQUEST(11);
```

Comment:
We shouldn't add an additional code here as this won't be compatible with other implementations. [Apache Thrift](https://github.com/apache/thrift/blob/master/lib/cpp/src/thrift/TApplicationException.h) stops at 10, but [Facebook Thrift](https://github.com/facebook/fbthrift/blob/master/thrift/lib/cpp/TApplicationException.h) adds 11-13. Probably best to use `PROTOCOL_ERROR`. @dain thoughts?

Reply:
I didn't know this enum is the part of the Thrift standard. I don't think it makes sense to break compatibility for this error. Let me use `PROTOCOL_ERROR` instead.

Reply:
I agree adding new codes is dangerous for future comparability.  Also no other client will support them.